catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -342,10 +418,12 @@ private QueryRequestImpl createQueryRequest(DeleteRequest deleteRequest) {
                 .collect(Collectors.toList());
 
         QueryImpl queryImpl =
-                new QueryImpl(queryOperations.getFilterWithAdditionalFilters(idFilters));
-        queryImpl.setStartIndex(1);
-        queryImpl.setPageSize(deleteRequest.getAttributeValues()
-                .size());
+                new QueryImpl(queryOperations.getFilterWithAdditionalFilters(idFilters),
+                        1,  /* start index */
+                        0,  /* page size */
+                        SortBy.NATURAL_ORDER,
+                        false, /* total result count */
+                        0   /* timeout */);
```

Comment:
Query requests provide a timeout but update and delete requests do not.  We should provide a timeout for these operation in the catalog framework.  I am fine with making that a separate ticket and leaving it 0 for now.